testApp/src/Components/NavBar.svelte

```diff
@@ -0,0 +1,83 @@
+<!-- NavBar.svelte -->
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+    import Sidebar from './Sidebar.svelte';
+
+    const dispatch = createEventDispatcher();
+    let logoPath = "/images/movieshowcase.jpeg";
+    let sidebarOpen = false;
+
+    function selectFranchise(id) {
+        if (id === 17) {
+            logoPath = "/images/movieshowcase.jpeg";
+        } else if (id === 4) {
+            logoPath = "/images/marvel_logo.jpeg";
+        }
+        dispatch('franchiseSelected', { id });
```

Comment:
You seem to be mixing double and single quotes for your string declarations. We usually always use single quotes for this.

Reply:
Thanks for the tip i'll keep it in mind! and I've implemented this ofc.

https://github.com/BowenGeense/MovieShowcase/pull/3/files#diff-1dfdde75a87aff5b0adc375b742824f63147ae1cf387f0711ab5df598f638c97